### DISCUSSION TITLE
Zero-Shot classifier TranslatorFactory

### DISCUSSION THREAD

**Discussion by lordofthejars** [deepjavalibrary/djl#3568] [General]
Hi, I have not started the development yet but, I prefer to ask before starting the development as maybe you've already done this.

I am trying to develop an example with zero/shot classifier example using the https://huggingface.co/MoritzLaurer/deberta-v3-base-zeroshot-v1.1-all-33 model, I think there is no default translator developed yet in the https://github.com/deepjavalibrary/djl/tree/master/extensions/tokenizers/src/main/java/ai/djl/huggingface/translator package.

My idea, if not supported, would be to send the support for it.

Thank you very much.

**Comment by frankfliu** [comment]
@lordofthejars 
We don't have `ZeroShotClassificationTranslator` yet, will add one soon.

**Reply by lordofthejars** [reply]
oh great to hear. I'll keep an eye then

**Comment by frankfliu** [comment]
@lordofthejars 

https://github.com/deepjavalibrary/djl/pull/3579
